Sources/SwiftCurrent_SwiftUI/Views/ModifiedWorkflowView.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  ModifiedWorkflowView.swift
+//  SwiftCurrent
+//
+//  Created by Tyler Thompson on 7/20/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+//  swiftlint:disable file_types_order
+
+import SwiftUI
+import SwiftCurrent
+
+@available(iOS 14.0, macOS 11, tvOS 14.0, watchOS 7.0, *)
+public struct ModifiedWorkflowView<Args, Wrapped: View, Content: View>: View {
```

Comment:
I missed it.  I'm not sure how this ended up on the screen.  Where did this body get put on the view tree?

Reply:
`thenProceed(with:` from a `WorkflowView` creates on of these bad boys.

Reply:
That instantiates a `ModifiedWorkflowView`.  I meant my question more towards, I don't see how the body of a `ModifiedWorkflowView` makes it up to the `ContentView` that created the `WorkflowView`.  `WorkflowView` always has a `fatalError` in its body, so I'm missing the connection between `ContentView { var body: some View }` and `ModifiedWorkflowView` body.

Reply:
Wait ok, I think it's clicking.  The `thenProceed(with:` returns a `ModifiedWorkflowView`, thus replacing my `WorkflowView` with a `ModifiedWorkflowView` and 💥  the `WorkflowView` is not on the view graph.

Reply:
Winner winner chicken dinner. That's precisely it.